.ci_support/win_64_.yaml

```diff
@@ -24,15 +24,6 @@ libnetcdf:
 - 4.8.1
 pcre:
 - '8'
-pin_run_as_build:
-  curl:
-    max_pin: x
-  fftw:
-    max_pin: x
-  gdal:
-    max_pin: x.x
-  zlib:
-    max_pin: x.x
```

Comment:
@seisman, not sure why these were removed, but it might be causing the issue at https://github.com/GenericMappingTools/pygmt/issues/2215. Did something change in conda-forge recently?

Reply:
Looking at https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/3410, it seems like we may need to specify [`run_exports`](https://conda-forge.org/docs/maintainer/pinning_deps.html#specifying-run-exports) in the `recipe/meta.yaml` file.